deploy/install.sh

```diff
@@ -4,7 +4,7 @@
 # Licensed under the MIT license.
```

Comment:
@mandarinamdar This update requires update to the reference architecture doc in the architecture center. Should we coordinate this release with the update to the reference architecture doc in arch center?